src/main/webapp/app/shared/badge/NotCuratableBadge.tsx

```diff
@@ -22,6 +24,10 @@ const NotCuratableBadge: React.FunctionComponent<INotCuratableBadgeProps> = ({ m
     const mutationStrings: [string[], string[]] = [[], []];
     for (const name of mutationName.split(',')) {
       const trimmedName = name.trim();
+      if (Object.values(CategoricalAlterationType).includes(parseAlterationName(trimmedName)[0].alteration as CategoricalAlterationType)) {
```

Comment:
Currently, we for non-string mutations, only `Oncogenic Mutations` is not curatable. See util method: `isMutationEffectCuratable`. Maybe move the excludedMutations arrays out to a constants, so you can use it here?

Reply:
Ah gotcha, looking at the method, I could just use it directly